### ISSUE TITLE
Syntax to configure rules within categories

### ISSUE THREAD

**Issue by x80486** [pmd/pmd#4784]
## Description

Currently, configuring rules involves re-declaring the rule with the reference and rule name. This can be cumbersome — as it's XML in general. I think enhancing the XML syntax to allow the configuration of the actual rules directly inside the `<rule>` tag could make this clear.

## Describe the Solution You Would Like

Something like this would be ideal:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<ruleset ...>
  <rule ref="category/java/codestyle.xml">
    <configure name="ThisOne">
      <property name="other" value="" />
      <property name="another" value="" />
    </configure>
    <exclude name="SomeOther" />
  </rule>
</ruleset>
```

This way it's easier to include all the rules from a given rule set, and be able to configure just the few that match the project's guidelines.

## Describe Alternatives You Have Considered

N/A

## Additional Context

N/A
